PXWeb/Code/Api2/Cache/CachedResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace PxWeb.Code.Api2.Cache
+{
+    public class CachedResponse
+    {
+        public string content { get; set; }
```

Comment:
store the content in a byte array instead to make sure we don not get any sideeffects of string magic, since the content might be binary.